### ISSUE TITLE
cannot multi-select playlist items if the current page is scrolled off

### ISSUE THREAD

**Issue by gnowxilef** [u-wave/web#648]
it reverts to allowing a single item to be selected (and then you can only multi-select on the same page)

**Comment by gnowxilef** [comment]
actually, it might be that it is a bit buggy. it sometimes works and sometimes does not.

**Comment by goto-bus-stop** [comment]
Yeah you're right, it is indeed a bit buggy, I'm not sure why. Thanks for opening an issue for it 🙇‍♀️

**Comment by goto-bus-stop** [comment]
Ah, I think it happens when a new playlist page is autoloaded while you scroll. That replaces the playlist items array, which [clears the selection](https://github.com/u-wave/web/blob/504767d7ffcd3d927187d1003a6cb298d532ea93/src/components/MediaList/index.js#L33-L37).

**Comment by goto-bus-stop** [comment]
Fixed in #684, thanks!

**Comment by gnowxilef** [comment]
seems like it still does it.

**Comment by gnowxilef** [comment]
i'm noticing that it happens semi-infrequently, but it still seems to occur.

**Comment by gnowxilef** [comment]
i noticed that it seems to happen if i scroll and don't click within some number of time.

**Comment by goto-bus-stop** [comment]
I just ran into this again and I think it happens primarily with the shift-click range selection, so if you select 1 thing and scroll for a while and shift-select another it doesn't remember the range.
